src/System.Diagnostics.Debug/tests/DebugTestsUsingListeners.cs

```diff
@@ -0,0 +1,199 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Reflection;
+using Xunit;
+
+namespace System.Diagnostics.Tests
+{
+    // These tests test the static Debug class. They cannot be run in parallel
+    // Shows that Debug behaves identically on all of the test cases in DebugTestsNoListeners even when DebugUsesTraceListeners is true.
+    [Collection("System.Diagnostics.Debug")]
```

Comment:
`DebugTestsUsingListeners` runs all `DebugTestsNoListeners` in addition to the tests written below.